osfoffline/tasks/operations.py

```diff
@@ -20,6 +20,12 @@
 logger = logging.getLogger(__name__)
 
 
+def get_node_title(node_id):
```

Comment:
This functions seems to duplicate operations that may already be implemented. (and guarantees an extra server hit that may not always be necessary)

The OperationContext object should (usually) provide a way to get data about a node from the OSF, via the `self.remote` attribute. For any operation where there is a database entry (and the node is known on the server), you should be able to write something like `self.remote.title` to get the node title, without needing this function.

See:
https://github.com/abought/OSF-Offline/blob/cd44cf188deffadaa62795507f8f888102498f6d/osfoffline/tasks/operations.py#L72-L72
https://github.com/abought/OSF-Offline/blob/167a737e9f3442d7baf6c2faf058d9b46eebd8d1/osfoffline/client/osf.py#L55-L55

If you do need a separate function, consider implementing the server call in `osfclient.py`, so that related code is kept together in a logical way.